apps/condo/domains/banking/components/BankAccountInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import { Col, Row } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import React, { CSSProperties } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography } from '@open-condo/ui'
+
+const DISPLAYED_BANK_ACCOUNT = ['tin', 'routingNumber', 'number']
+
+const VERTICAL_GUTTER: [Gutter, Gutter] = [0, 24]
+
+const BankAccountRow = ({ bankAccountName, bankAccountValue, title = false }) => {
+    const intl = useIntl()
+    const recipientTitle = intl.formatMessage({ id: 'pages.condo.settings.bankAccount.' + bankAccountName })
```

Comment:
IMO `bankAccountName` property naming is non-obvious

Reply:
done